common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/SecurityCommonAutoConfigurationProperties.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.security.common;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * {@code SecurityCommonAutoConfiguration} properties.
+ *
+ * @author Christian Tzolov
+ * @author Artem Bilan
+ */
+@ConfigurationProperties("spring.cloud.security.security")
```

Comment:
Duplication: `security.security`.
I thought we agreed about `stream.security` 😄

Reply:
Huh? Did you miss this somehow?

Thanks

Reply:
uh, i did apparently :(